cobalt/devinfra/kokoro/build/linux/package.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+"""Packages Cobalt for Linux."""
+
+import os
+import shutil
+import sys
+import tempfile
+
+out_dir = sys.argv[1]
+package_dir = sys.argv[2]
+
+with tempfile.TemporaryDirectory() as tmp_dir:
+  base_dir = os.path.join(tmp_dir, 'cobalt-linux')
```

Comment:
make `cobalt-linux` into an argument, don't hardcode it. We have 2 different linux configs at least.

Reply:
I believe the other Linux config was deleted by #4749.
Also note Chromium uses the same name (hardcoded?) here:
https://chromium.googlesource.com/chromium/src/+/114.0.5735.358/infra/archive_config/android-archive-rel.json
https://chromium.googlesource.com/chromium/src/+/114.0.5735.358/infra/archive_config/android-arm64-archive-rel.json
I believe there's no conflict there or here.